sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Primitives/ServiceBusMessageActions.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Messaging.ServiceBus;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    /// <summary>
+    /// Represents the set of message actions that can be performed on a <see cref="ServiceBusReceivedMessage"/>.
+    /// </summary>
+    public class ServiceBusMessageActions
```

Comment:
Should we add an inheritance relationship between ServiceBusMessageActions and SessionActions?

Reply:
We already have a bunch of if-elses we would need to add another one for session.